src/V2World/Provinces/Province.cpp

```diff
@@ -36,6 +35,26 @@ int Vic2::Province::getLiteracyWeightedPopulation(const std::optional<std::strin
 	return totalPopulation;
 }
 
+double Vic2::Province::getPercentageWithCulture(const std::string& culture) const
```

Comment:
Could you use the `getPercentageWithCultures(...)` function with a set of just one culture instead?